src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPlugin.kt

```diff
@@ -193,4 +192,22 @@ class NexusPublishPlugin : Plugin<Project> {
     private inline fun <reified T : Task> TaskContainer.withName(name: String): TaskProvider<T> {
         return withType<T>().named(name)
     }
+
+    private fun createSimplifiedCloseAndReleaseTask(rootProject: Project, extension: NexusPublishExtension) {
+        if (extension.repositories.isNotEmpty()) {
```

Comment:
❓ Why the condition? Couldn't we just always create it?

Reply:
We could. However, currently we create the other tasks [only](https://github.com/gradle-nexus/publish-plugin/blob/d4b239ea447132a857d9dcde24be0cf2e658d596/src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPlugin.kt#L71-L78) if there is any Nexus repo defined. Therefor, that task would do nothing (and its description would have to be funny ;-) ).